### ISSUE TITLE
Downthemall replacement?

### ISSUE THREAD

**Issue by DrummerHead** [inbasic/bulk-media-downloader#24]
I just want to make you aware that right now your project is being recommended as a replacement to http://www.downthemall.net/ in the Firefox legacy extensions page (within the browser's options; about:addons last option).

From installing bulk-media-downloader addon and looking at its functionality, it doesn't seem you're solving the same problem that downthemall solves.

Perhaps you can consider adding a comment in [the description Firefox links to](https://addons.mozilla.org/en-US/firefox/addon/bulk-media-downloader/) stating you're not exactly the same as downthemall, or perhaps take this as an opportunity to do some [scope creep](https://en.wikipedia.org/wiki/Scope_creep)ing and actually adopt downthemall functionality.

Cheers!

**Comment by jamminjames** [comment]
I agree. If you could adopt the AntiContainer functionality of DownThemAll!, which allows the user to download resources that are normally hidden behind container pages, that would be great. I can find no other downloader extension that does this.
